src/main/kotlin/org/rust/ide/typing/RustEnterInStringLiteralHandler.kt

```diff
@@ -59,4 +60,8 @@ class RustEnterInStringLiteralHandler : EnterHandlerDelegateAdapter() {
             else -> Result.Continue
         }
     }
+
+    private fun isValidOffset(offset: Int, text: CharSequence): Boolean {
+        return offset >= 0 && offset < text.length
```

Comment:
I like to write similar comparisons as `0 <= offset && offset < text.lenght`, because that way `<` are aligned with the usual direction on the number line. Have I already said this before?